src/sass/components/_Alert.scss

```diff
@@ -5,19 +5,48 @@
   a.usa-button {
     color: $color-white;
   }
-  .alert-check {
-    border-radius: 50%;
-    display: inline-block;
-    margin-right: 0.5em;
-    margin-top: -1em;
-    margin-bottom: -0.3em;
-    height: 30px;
+  &.usa-alert-info,&.usa-alert-warning, &.usa-alert-error, &.usa-alert-success {
+    background-image: none;
+  }
+  .AlertImage {
+    font-size: 30px;
+    line-height: 25px;
+    padding-top: 2px;
+    color: #fff;
+    position: absolute;
     width: 30px;
-    border: 2px solid $color-base;
-    background-color: $color-base;
-    background-image: url("/img/check.png");
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: 13px auto;
+    height: 30px;
+    border-radius:50%;
+    background-color: #212121;
+    font-weight: 700;
+    text-align: center;
   }
+
+}
+.usa-alert-success > .AlertImage {
+  background-image: url('../img/correct8.png');
+  background-image: url('../img/correct8.svg');
+  background-repeat: no-repeat;
+  background-size: 30px;
+}
+
+.usa-alert-error > .AlertImage {
+  background-image: url('../img/close-white.png');
+  background-image: url('../img/close-white.svg');
+  background-repeat: no-repeat;
+  background-size: 16px;
+  background-position: 7px;
+  background-color: $color-secondary;
```

Comment:
We can remove this line to keep all the `background-color` for icons in alerts consistent.